lib/cc/analyzer/filesystem.rb

```diff
@@ -47,6 +37,14 @@ def files_matching(globs)
 
       private
 
+      def file_paths
+        @file_paths ||= Dir.chdir(@root) do
+          `find . -type f -print0`.strip.split("\0").map do |path|
+            path.sub(/^\.//, "")
```

Comment:
There may be caveats, see http://stackoverflow.com/questions/2596462/how-to-strip-leading-in-unix-find/2596474#2596474 for details.